internal/tiltfile/files.go

```diff
@@ -181,8 +181,10 @@ func (s *tiltfileState) recordConfigFile(f string) {
 	s.configFiles = sliceutils.AppendWithoutDupes(s.configFiles, f)
 }
 
-func (s *tiltfileState) readFile(p localPath) ([]byte, error) {
-	s.recordConfigFile(p.path)
+func (s *tiltfileState) readFile(p localPath, recordConfigFile bool) ([]byte, error) {
```

Comment:
this seems like a weird api? if you didn't want to record the config file, wouldn't you just call ioutil.ReadFile directly?